examples/__tests__/test-state-migration.ava.js

```diff
@@ -0,0 +1,36 @@
+import { Worker } from "near-workspaces";
+import test from "ava";
+
+test.beforeEach(async (t) => {
+    const worker = await Worker.init();
+    const root = worker.rootAccount;
+    const contract = await root.devDeploy("./build/state-migration.wasm");
```

Comment:
To make this example align with real world workflow, let's make two contracts: `old-message.wasm` and `state-migration.wasm`. 
In `old-message.wasm`, it has `OldMessageFormat`, `addMessage({ message }: {message: OldMessageFormat})` and `countMessages`. 
And `state-migration.wasm` is just built from current `examples/src/state-migration.ts`.

In the test, developer first deploy `old-message.wasm`, add a couple of old messages. Then he deployed `state-migration.wasm`, call `migrateState` and `countMessages`.

Reply:
Good point, refactored to have 2 contracts.

Reply:
Great! Test looks self-explanatory!